File: 2017ACMQualifiers/you_can_say_11.py
import fileinput
isMult = "is a multiple of 11."
isNotMult = "is not a multiple of 11."

for line in fileinput.input():
    nospace = line.strip(' \n\t')
    intRep = int(nospace)
    if intRep != 0:
        if intRep % 11 == 0:
            print(nospace, isMult)
        else:
            print(nospace, isNotMult)

File: 2017ACMQualifiers/you_can_say_11.cpp
#include<iostream>
#include<string>

using namespace std;

int main(){
    string isMult = " is a multiple of 11.";
    string isNotMult = " is not a multiple of 11.";
    int num;
    cin >> num;
    while( num ){
        if( num % 11 == 0 )
            cout << num << isMult << endl;
        else
            cout << num << isNotMult << endl;
        cin >> num;
        //cout << "Got: " << num << endl;
    }
    return 0;
}
